src/frontend/src/models/PersonnelRequest.ts

```diff
@@ -5,7 +5,13 @@ import Position from './Position';
 import Project from './Project';
 import Comment from './Comment';
 
-export type RequestState = 'Created' | 'Submitted' | 'Approved' | 'Rejected' | 'Provisioned';
+export enum RequestState {
```

Comment:
Why was this changed to an enum?

Reply:
As of right now the backend returns a number between 0 and 3 for status, but this is maybe a bug?

Reply:
@HansDahle Could the request state be the string value of the enum, instead of int? Easier for both the resource app and other consumers to reason about a human readable string value vs just a random number

Reply:
yes, guess a converter is forgotten

Reply:
Changed back to type